chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/termination/JTerminationIterations.java

```diff
@@ -0,0 +1,66 @@
+package com.griddynamics.jagger.user.test.configurations.termination;
+
+/**
+ * Limits the number of execution iterations of a test.
```

Comment:
Test with such termination strategy will last till defined number of requests are executed